### PR TITLE
Fixed suggestions about getting ES8336 quirk default.

### PR DESCRIPTION
**Pull request by nnseva**
The documentation was wrong about getting the quirk default. I've fixed these details.

### CODE REVIEW

**Review by lgirdwood — COMMENTED** [review]
@plbossart @ujfalusi pls review.

**Review by cujomalainey — CHANGES_REQUESTED** [review]
commit message is empty and unhelpful

also probably missing sign-off

**Comment by deb-intel** [comment]
> commit message is empty and unhelpful
> 
> also probably missing sign-off

@cujomalainey, to whom are you addressing?

**Comment by cujomalainey** [comment]
> > commit message is empty and unhelpful
> > 
> > also probably missing sign-off
> 
> @cujomalainey, to whom are you addressing? 

The author, you cannot comment on commit messages directly hence the un-anchored review comment

File: getting_started/intel_debug/suggestions.rst
Comment (marc-hb) [inline getting_started/intel_debug/suggestions.rst:308]: This is invalid for some reason:

https://github.com/thesofproject/sof-docs/actions/runs/6297068973/job/17095424648?pr=477

```
/home/runner/work/sof-docs/sof-docs/getting_started/intel_debug/suggestions.rst:300: WARNING: Pygments lexer name 'txt' is not known
/home/runner/work/sof-docs/sof-docs/getting_started/intel_debug/suggestions.rst:308: WARNING: Pygments lexer name 'txt' is not known

```

**Review by marc-hb — CHANGES_REQUESTED** [review]
Do not add warnings.

File: getting_started/intel_debug/suggestions.rst
Comment (marc-hb) [inline getting_started/intel_debug/suggestions.rst:298]: There is `/var/log/dmesg` file on Fedora or Arch Linux, it's distro-specific.

Don't recommend the `dmesg` command either because it's a ring buffer that can run out.

There are countless pages on the Internet that explain what each distro does with kernel logs, just let users find that themselves:
```suggestion
in the kernel logs in a line that looks like:
```

File: getting_started/intel_debug/suggestions.rst
Comment (nnseva) [reply getting_started/intel_debug/suggestions.rst]: the code-block fixed to `none`.

File: getting_started/intel_debug/suggestions.rst
Comment (nnseva) [reply getting_started/intel_debug/suggestions.rst]: fixed

**Comment by nnseva** [comment]
> commit message is empty and unhelpful
> 
> also probably missing sign-off

Tried to explain the change. Added my sign-off.

**Review by marc-hb — DISMISSED** [review]

**Comment by marc-hb** [comment]
The commit message body in https://github.com/thesofproject/sof-docs/pull/477/commits/d555d7a7646 is still empty. Not a problem for me because the commit message title looks very good to me now and enough for this small fix but there are some people and some policies who insist on non-empty commit messages :-)

**Review by lgirdwood — APPROVED** [review]

**Comment by btian1** [comment]
fixed --> fix?

**Comment by cujomalainey** [comment]
> some people and some policies who insist on non-empty commit messages :-)

Yes but I won't argue to block it if everyone else is fine

**Review by lgirdwood — APPROVED** [review]

**Review by deb-intel — CHANGES_REQUESTED** [review]
Please incorporate the minor grammatical edits at your convenience.

**Review by btian1 — APPROVED** [review]

File: getting_started/intel_debug/suggestions.rst
Comment (nnseva) [reply getting_started/intel_debug/suggestions.rst]: fixed

File: getting_started/intel_debug/suggestions.rst
Comment (nnseva) [reply getting_started/intel_debug/suggestions.rst]: fixed

File: getting_started/intel_debug/suggestions.rst
Comment (nnseva) [reply getting_started/intel_debug/suggestions.rst]: fixed

File: getting_started/intel_debug/suggestions.rst
Comment (nnseva) [reply getting_started/intel_debug/suggestions.rst]: fixed

File: getting_started/intel_debug/suggestions.rst
Comment (nnseva) [reply getting_started/intel_debug/suggestions.rst]: fixed

**Review by marc-hb — COMMENTED** [review]
Please git squash @deb-intel 's fixes and force push a single commit.

We don't use GitHub "the GitHub way" in this project, we use GitHub in the older, more traditional git way where each commit in a PR stands alone / is its own thing.

For more context:
https://github.com/zephyrproject-rtos/zephyr/issues/39194
https://gitlab.com/gitlab-org/gitlab/-/issues/24096

PS: yes the maintainer can "squash and merge" at the end but we usually don't do that, see above why.

**Comment by nnseva** [comment]
> Please git squash @deb-intel 's fixes and force push a single commit.
done

**Review by deb-intel — APPROVED** [review]
Approved with integrated change requests.

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
All reviewer comments now addressed.

**Pull request merged**
